src/components/atoms/Timeline/index.stories.tsx

```diff
@@ -29,3 +30,19 @@ export const DefaultRange: Story<Props> = () => (
     renderKnob={() => <Icon icon="ellipse" alt="ellipse" size={25} />}
   />
 );
+
+export const Movable: Story<Props> = () => {
+  const [currentTime, setCurrentTime] = useState(() => Date.now() + 3600000);
+  const onClick = useCallback(setCurrentTime, [setCurrentTime]);
+  const onDrag = useCallback(setCurrentTime, [setCurrentTime]);
```

Comment:
Can you explain why you are wrapping the setCurrentTime in useCallbacks? Is it just for simpler typings as a Timeline prop? Or some other reason?

Reply:
I know this is a stories file, so if this is just to make sure the example is doing something specific, and not how it'll be used actually inside Re:Earth, you can resolve this message!